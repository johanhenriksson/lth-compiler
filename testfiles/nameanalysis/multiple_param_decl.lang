int main(int argc, int argc) { // error
}
