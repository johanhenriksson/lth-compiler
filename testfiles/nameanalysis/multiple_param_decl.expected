Error at line 1,20: symbol 'argc' is already declared!
